dpgen/generator/run.py

```diff
@@ -1295,7 +1426,13 @@ def _make_fp_vasp_inner (modd_path,
         cwd = os.getcwd()
         for ii in fp_tasks:
             os.chdir(ii)
-            dump_to_poscar('conf.dump', 'POSCAR', type_map)
+            if model_devi_engine == "lammps":
+                dump_to_poscar('conf.dump', 'POSCAR', type_map, fmt = "lammps/dump")
+            elif model_devi_engine == "gromacs":
+                # dump_to_poscar('conf.dump', 'POSCAR', type_map, fmt = "gromacs/gro")
+                dump_to_deepmd_raw('conf.dump', 'deepmd.raw', type_map, fmt = 'gromacs/gro')
```

Comment:
This `conf.dump` is seemed to be a symbolic link of `.gromacstrj` file. Does dpdata support to read it?

Reply:
Actually,  ".gromacstrj" format  is  just a name convention. In codes, dpdata load this file by fmt gromacs/gro.